reference/api/multi_index.md

```diff
@@ -0,0 +1,121 @@
+# Multi-index search
+
+The `/multi-search` route allows you to perform multiple search queries on one or more indexes by bundling them into a single request.
+
+## POST route
+
+<RouteHighlighter method="POST" route="/multi-search"/>
+
+### Body
+
+| Name          | Type             | Description                                                                                                                                          |
+| :------------ | :--------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`queries`** | Array of objects | Contains the list of search queries to perform. The [`indexUid`](#search-parameters) search parameter is required, all other parameters are optional |
+
+#### Search parameters
+
+| Search parameter                                                                | Type             | Default value | Description                                                               |
+| :------------------------------------------------------------------------------ | :--------------- | :------------ | :------------------------------------------------------------------------ |
+| **[`indexUid`](/reference/api/search.md#query-q)**  *                           | String           | N/A           | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
+| **[`q`](/reference/api/search.md#query-q)**                                     | String           | `""`          | Query string                                                              |
```

Comment:
Do you know if we can reuse components as we do on the openAPI spec so we don't have to rewrite/copy&paste at multiple places within the documentation when we just plug existing parameters on resources like here?

Reply:
Atm, no. But maybe we could with the new site 🤔